### DISCUSSION TITLE
Add SwitchBot to Button integration

### DISCUSSION THREAD

**Discussion by damiano75** [home-assistant/frontend#10919] [Dashboard] [answered]
I have two switchbots that work like buttons, there isn't any state, just a pressed button, so I think it could be a good idea to have a button entity for these switchbots

**Comment by frenck** [comment, marked as answer]
So a couple of things:

- I think you misunderstand what button entities do. Button entities send a press event to a device or service, not the other way around. So, those are not a fit.
- This isn't a frontend feature request, but a request for a new feature for an integration. For these purposes, we have the Feature Request category on our community forum.
  https://community.home-assistant.io/c/feature-requests/13
